nservicebus/messaging/databus.md

```diff
@@ -68,7 +68,22 @@ snippet:MessageWithLargePayloadUsingConvention
 
 NServiceBus DataBus implementations currently behave differently with regard to cleanup of physical attachments used to transfer data properties. `FileShareDataBus` **does not** remove physical attachments once the message is gone. `AzureDataBus` **does** remove Azure storage blobs used for physical attachments once the message is gone.
 
+### Custom cleanup strategy
 
+NServiceBus cannot tell when a file should be deleted, as removing the file depends on your functional needs. There are a large number of scenarios where removing the file can cause a problem. 
+
+- The message can be (functionally) deferred so that the file will be processed later, or it is passed along to another handler. When the file is removed upon committing the transaction, the deferred or passed on message cannot process the file anymore.
+- The file system isn’t transactional. Transactions spawned by NServiceBus might fail, but the file system cannot participate in them, resulting in a deleted file even when the transaction is aborted.
```

Comment:
How about:
- The file system does not participate in distributed transactions. If for some reason, the message handler throws an exception and the transaction rolls back, the file delete operation on the attachment cannot be rolled back. Therefore, when the message is retried, the attachment will no longer be present causing additional problems.